packages/ariakit/src/checkbox/__examples__/checkbox-controlled/style.css

```
@@ -0,0 +1,7 @@
+.label {
+  @apply flex flex-row items-center gap-2;
+}
+
+.checkbox {
+  @apply ariakit-focus-visible:ariakit-outline;
+}
```

Does it work if we just import the base checkbox styles?
```suggestion
@import url("../checkbox/style.css");
```